docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst

```diff
@@ -487,7 +487,7 @@ Specific outputs for specific models are specified in the following section.
     - None
     - * Deaths 
       * YLLs
-      * YLDs
+      * YLDs (including YLDs due to "all causes")
```

Comment:
```suggestion
      * YLDs (including YLDs due to "other causes")
```

Or maybe "background morbidity" since you changed the title of that page?

Reply:
This is actually intentionally written as "all" causes because of the detail in the vivarium behavior for calculating all cause and cause-specific YLDs differently, as shown in this screenshot from this page: https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_features/other_causes_ylds/index.html

![image](https://github.com/ihmeuw/vivarium_research/assets/54918472/79298860-7ec4-458a-a8f4-0073c75beea1)

Reply:
Ah yep, I did not interpret it that way. This is better!